lib/parser.js

```diff
@@ -154,6 +154,7 @@ async function customDocumentOperations(js, asyncapiYAMLorJSON, initialFormat, o
   
   validateChannelParams(js, asyncapiYAMLorJSON, initialFormat);
 
+  /* eslint-disable security/detect-object-injection */
```

Comment:
`js.channels` can be converted into Map -> https://github.com/asyncapi/parser-js/pull/105/files#diff-c3d5db07c30dd8c16e1da041b6f90850R115

of course depends what part is a problem for the linter

Reply:
The problem is with retrieving value by `channelName` and `opName` name. As you see, this variable will be in 100% strings.

Reply:
And as I remember `forEach` method from ES6 Map cannot handle async functions, and in this function we must handle async conversion of Messages.

Reply:
you are right about `forEach` but then you can just use `for of`

thing is that we should not assume that what we have now (100% string) will always be like that if the future, so better just to fix than ignore